src/test/java/io/vinyldns/java/VinylDNSClientTest.java

```diff
@@ -772,6 +772,41 @@ public void createBatchChangeSuccess() {
     assertEquals(vinylDNSBatchResponse.getStatusCode(), 202);
   }
 
+  @Test
+  public void listRecordSetChangeSuccess() {
+    String response = client.gson.toJson(listRecordSetChangesResponse);
+
+    wireMockServer.stubFor(
+            get(urlEqualTo("/zones/" + zoneId + "/recordsetchange"))
```

Comment:
URL needs to be `/recordsetchanges` for the route to match the `path` in `VinylDNSClientImpl.java` and for the tests to pass.